etc/gattaca01.cfg

```diff
@@ -1 +1,6 @@
 SCR_LFS='scr01'
+ROOTDIR="/groups/"
+LABDIR="/groups/umcg-lab/scr01/"
+SEQDIR="${LABDIR}/sequencers/"
+MISSINGSAMPLESHEETSDIR=${LABDIR}/logs/
```

Comment:
Where is this used? And if necessary, why is it not called something like ${LOG_DIR}?